supplementary_style_guide/glossary_terms_conventions/product_conventions/eap-conventions/eap-terms-and-usage--j.adoc

```diff
@@ -0,0 +1,222 @@
+= J
+// Should this term be added to the general "j" topic?
+[discrete]
+== Java
+
+*Description* A class-based, object-oriented programming language.
+
+*Use it?* Yes
+
+*Incorrect forms* java (lower-case)
+
+*See*
+
+[discrete]
+== jaxrs
+
+*Description* The JBoss EAP subsystem that enables the deployment and functionality of RESTful web services through the Java API for RESTful Web Services (JAX-RS).
+
+*Use it?* Yes
+
+*Incorrect forms* jax-rs, JAXRS, JAX-RS
+
+*See* JAX-RS
+
+[discrete]
+== JAX-RS
+
+*Description* Heading format for the jaxrs subsystem.
+
+*Use it?* Yes
+
+*Incorrect forms* JAXRS
+
+*See* jaxrs
+
+//Should "JBoss AMQ" be moved to the AMQ module?
+[discrete]
+== JBoss AMQ
+
+*Description* Former name of Red Hat's messaging queue product.  Product has been renamed to "Red Hat AMQ"
+
+*Use it?* No
+
+*Incorrect forms*
+
+*See* Red Hat AMQ
+
+[discrete]
+== JBoss EAP messaging
+
+*Description* Acceptable term for the built-in messaging system in JBoss EAP.
+
+*Use it?* Yes
+
+*Incorrect forms* ActiveMQ, ActiveMQ Artemis
+
+*See* "built-in messaging", "JBoss EAP built-in messaging", "messaging subsystem"
```

Comment:
Typically, the comma goes inside the closing quotation mark.

Reply:
This is now fixed based on updates to the jboss-eap.adoc file in https://github.com/redhat-documentation/supplementary-style-guide/pull/97. Commas removed from links.